prelude/prelude.purs

```diff
@@ -120,6 +120,9 @@ module Prelude
   class Functor f where
     (<$>) :: forall a b. (a -> b) -> f a -> f b
 
+  fmap :: forall f a b. (Functor f) => (a -> b) -> f a -> f b
+  fmap = (<$>)
```

Comment:
I don't like the name `fmap` for this function. If we have it, it should just be `map`.